app/indexers/object_indexer.rb

```diff
@@ -194,20 +194,33 @@ def file_set_images(size: Settings.thumbnails["basic"]["size"],
       object.file_sets.map do |file_set|
         file = file_set.files.first
         next unless file
-        Riiif::Engine.routes.url_helpers.image_url(
-          file.id,
-          size: size,
-          region: region,
-          only_path: true
-        )
+        if Rails.configuration.try(:external_iiif_url)
+          rotation = "0"
+          quality = "default"
+          format = "jpg"
+          "#{Rails.configuration.external_iiif_url}#{CGI.escape(file.id)}/#{region}/#{size}/#{rotation}/#{quality}.#{format}"
```

Comment:
Could you do `rubocop:disable` for this line instead of excluding the whole file?